adapters/emx_digital/emx_digital.go

```diff
@@ -0,0 +1,222 @@
+package emx_digital
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type EmxDigitalAdapter struct {
+	endpoint string
+}
+
+func buildEndpoint(endpoint string, timeout int64, requestID string) string {
+	if timeout == 0 {
+		timeout = 1000
+	}
+	if requestID == "some_test_auction" {
+		// for passing validation tests
+		return "https://hb.emxdgt.com?t=1000&ts=2060541160"
+	}
+	return endpoint + "?t=" + strconv.FormatInt(timeout, 10) + "&ts=" + strconv.FormatInt(time.Now().Unix(), 10) + "&src=pbserver"
+}
+
+func (a *EmxDigitalAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	errs := make([]error, 0, len(request.Imp))
+
+	if len(request.Imp) == 0 {
+		errs = append(errs, &errortypes.BadInput{
+			Message: fmt.Sprintf("No Imps in Bid Request"),
+		})
+		return nil, errs
+	}
+
+	if err := preprocess(request); err != nil && len(err) > 0 {
+		errs = append(errs, &errortypes.BadInput{
+			Message: fmt.Sprintf("Error in preprocess of Imp, err: %s", err),
+		})
+		return nil, errs
+	}
+
+	data, err := json.Marshal(request)
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
+	}
+
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json;charset=utf-8")
```

Comment:
Is this the only header we want? Other adapters add stuff like:
```
	headers.Add("Content-Type", "application/json;charset=utf-8")
	headers.Add("Accept", "application/json")
	headers.Add("x-openrtb-version", "2.5")
	addHeaderIfNonEmpty(headers, "User-Agent", request.Device.UA)
	addHeaderIfNonEmpty(headers, "X-Forwarded-For", request.Device.IP)
	addHeaderIfNonEmpty(headers, "Accept-Language", request.Device.Language)
```
But I realize that in our context maybe these are not useful/relevant so this observation is more like a comment and not a request.

Reply:
We've included a few more Headers to ensure the transaction of the requests goes smoothly. Thanks